src/main/resources/schema.sql

```diff
@@ -1,14 +1,24 @@
+DROP TABLE IF EXISTS mpa CASCADE;
```

Comment:
Эту конструкцию удаления всех таблиц можно сократить до:
```suggestion
DROP TABLE IF EXISTS 
mpa, film, film_user, genre, friendship, likes, genre_film, DIRECTORS, DIRECTORS_FILM CASCADE;
```